bin/usage.txt

```diff
@@ -6,23 +6,24 @@
   - You should tell where does azk binary is.
 
 Usage:
-  azk-benchmark [<azk_bin_path>] [--git-repo=<git-repo> --dest-path=<dest-path> --git-ref=<git-ref> --send] [--no-color] [--plain] [-v]...
+  azk-benchmark [<azkBinPath>] [--git-repo=<git-repo> --dest-path=<dest-path> --git-ref=<git-ref> --repeat=<repeat> --send] [--no-color] [--plain] [-v]...
```

Comment:
@saitodisse I know this PR is closed, but do you agree that `azk-bin-path` would be more appropriate? Once you have `git-repo`, `git-ref` and `dest-path`, I think the pattern should be kept.